### PR TITLE
Config changes to add SLOMeasurement- initial commit

### PR DESCRIPTION
**Pull request by vamossagar12**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by vamossagar12** [comment]
@wojtek-t This is the PR with the config change. One question that I had was that initially, NetworkProgrammingLatency was invoked only if the flag **PROMETHEUS_SCRAPE_KUBE_PROXY**  was set to true:

https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/load/config.yaml#L141-L146

With the blanket SLOMeasurement, it gets invoked every time. Do you think we need some changes to handle this?

File: clusterloader2/testing/load/config.yaml [hunk 1]
```diff
@@ -132,18 +132,10 @@ steps:
...
-      action: start
-      checkProbesReadyTimeout: {{$PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT}}
-      replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
-  {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
-  - Identifier: NetworkProgrammingLatency
-    Method: NetworkProgrammingLatency
-    Params:
-      action: start
-  {{end}}
+      action:start
```
Comment (wojtek-t) [inline clusterloader2/testing/load/config.yaml:138]: The other params (checkProbesReadyTimeout etc. are still needed - they are use by the measurement so we need to pass them.

**Comment by wojtek-t** [comment]
> @wojtek-t This is the PR with the config change. One question that I had was that initially, NetworkProgrammingLatency was invoked only if the flag PROMETHEUS_SCRAPE_KUBE_PROXY was set to true:

> https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/load/config.yaml#L141-L146

We should ideally try to solve that first - let me look into that a bit later today/tomorrow.

File: clusterloader2/testing/load/config.yaml [hunk 2]
```diff
@@ -854,16 +846,10 @@ steps:
...
     Params:
       action: gather
-  - Identifier: DnsLookupLatency
-    Method: DnsLookupLatency
-    Params:
-      action: gather
-  {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
-  - Identifier: NetworkProgrammingLatency
-    Method: NetworkProgrammingLatency
+  - Identifier: SLOMeasurement
```
Comment (wojtek-t) [inline clusterloader2/testing/load/config.yaml:849]: Actually - to no block on addressing the kube-proxy scraping, let's start differently:
- let's temporarily remove the NetworkProgrammingLatency from the SLO measurement (leave only DNSLookupLatency for now)
- and we will proceed with this PR (and address NetworkProgramminlatency separately)

**Comment by vamossagar12** [comment]
@wojtek-t , PR to remove network programming latency:
https://github.com/kubernetes/perf-tests/pull/1518

**Comment by wojtek-t** [comment]
@vamossagar12 - there seem to be a bug in:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/slos/slo_measurement.go#L65

it should be:
```
if !errList.IsEmpty() {
  return nil, errList
}
return summaries, nil
```

**Comment by wojtek-t** [comment]
Please fix (can be this PR).

**Pull request merged**